examples/optionals/as_bool.rs

```diff
@@ -0,0 +1,152 @@
+#![deny(warnings)]
+#![warn(rust_2018_idioms)]
+
+#[macro_use]
+extern crate log;
+
+use serde::Deserialize;
+use serde_this_or_that::as_opt_bool;
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Msg {
+    #[serde(deserialize_with = "as_opt_bool")]
+    pub timestamp: Option<bool>,
```

Comment:
I'm confused why you associate `timestamp` and `Option<bool>`. A timestamp is not boolean, so why not use something more typically boolean, like `truthy` or something that doesn't have any baggage like `foo`.

Reply:
done! agreed, my process was in cloning examples and this resulted in retaining the same named field `timestamp` for all examples, however I agree that it doesn't make sense in examples for `bool` fields. I renamed the field to `archived` so that we can be unambiguous.